antenna/app.py

```diff
@@ -59,7 +59,7 @@
 
 
 def count_sentry_scrub_error(msg):
-    METRICS.incr("app.sentry_scrub_error", value=1)
+    METRICS.incr("collector.sentry_scrub_error", value=1)
```

Comment:
This changes the key to `socorro.collector.sentry_scrub_error` which is more like the other things we're doing. I'll update the dashboard once this lands.